src/main/java/uk/gov/hmcts/reform/sscs/model/DescendantCaseData.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+
+import java.util.Map;
+
+@EqualsAndHashCode(callSuper = true)
+@Builder (builderMethodName = "DescendantCaseDataBuilder")
+public class DescendantCaseData extends uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData {
+
+    private Map<String, Object> caseFlags;
```

Comment:
Same comment applies - we can make this a Map<String, String> type.

Reply:
Yes we could do but this gives us greater flexibility and is more future proof.